flow/connectors/postgres/client.go

```diff
@@ -657,9 +657,16 @@ func (c *PostgresConnector) getCurrentLSN(ctx context.Context) (pglogrepl.LSN, e
 	var result pgtype.Text
 	err := row.Scan(&result)
 	if err != nil {
-		return 0, fmt.Errorf("error while running query: %w", err)
+		return 0, fmt.Errorf("error while running query for current LSN: %w", err)
 	}
-	return pglogrepl.ParseLSN(result.String)
+	if !result.Valid || result.String == "" {
+		return 0, fmt.Errorf("error while getting current LSN: query result is not valid text: %s", result.String)
```

Comment:
```suggestion
		return 0, fmt.Errorf("error while getting current LSN: no LSN available", result.String)
```
`result.String` will always be empty string here, can improve error message